src/documentation/transxchange_read.md

```diff
@@ -71,11 +71,11 @@ NTFS field | TransXChange element | Mapping rule/Comment
 --- | --- | ---
 line_id | *Services/Service/ServiceCode* | This field is prefixed.
 line_code | *Services/Service/Lines/Line/LineName* |
-line_name | *Services/Service/Description* | If no *Description* is found, the value is computed as "backward_line_name - forward_line_name" 
+line_name | *Services/Service/Description* | If no *Description* is found, then this field is computed using the name of the first associated Route in the forward direction. If several forward routes exist, the one with the smallest `route_id` is used.
 forward_line_name | *Services/Service/StandardService/Destination* | 
-forward_direction | *???* | 
+forward_direction |  | This field should have the same value as the `destination_id` of the first associated Route in the forward direction. If several forward routes exist, the one with the smallest `route_id` is used.
 backward_line_name | *Services/Service/StandardService/Origin* | 
-backward_direction | *???* | 
+backward_direction |  | This field should have the same value as the `destination_id` of the first associated Route in the backward direction. If several backward routes exist, the one with the smallest `route_id` is used.
```

Comment:
It must be origin_id I think

Reply:
@ArnaudOggy ot's OK like that, it's the destination of the backward route